src/components/HomeBlocks/ManageHomeBlocksModal.tsx

```diff
@@ -270,6 +270,7 @@ function SortableHomeBlock({
     transition,
     cursor: isDragging ? 'grabbing' : 'grab',
     zIndex: isDragging ? 1 : undefined,
+    touchAction: 'none',
```

Comment:
If we have any other draggable lists within a scroll container then we might need to do this. It prevents scroll while dragging.

Reply:
Can you please add it in the other places we use draggables? I believe you can do a find with `useSortable(...)` and add the `touchAction` to the custom styles